src/preflightCheck.js

```diff
@@ -1,7 +1,7 @@
 import { join } from 'path';
 import { transform } from 'babel-core';
 import { INLINE, RUNTIME, BUNDLED } from './constants.js';
-import classes from 'babel-plugin-transform-es2015-classes';
+import importHelperPlugin from './helperPlugin.js';
```

Comment:
I don't think it's necessary to import the plugin if we are just checking the helper